pkgs/development/libraries/onnxruntime/default.nix

```
@@ -55,6 +57,9 @@ stdenv.mkDerivation rec {
       compatibility.
     '';
     homepage = "https://github.com/microsoft/onnxruntime";
+    changelog = "https://github.com/microsoft/onnxruntime/releases";
```

let's refer to a specific changelog version:
```suggestion
    changelog = "https://github.com/microsoft/onnxruntime/releases/tag/v${version}";
```

---

but this is the meta section for for the whole package, wouldn't make sense to be pointing to latest so I don't have to bump it every release?